rero_ils/modules/holdings/cli.py

```diff
@@ -106,8 +117,11 @@ def create_patterns(infile, verbose, debug, lazy):
                 holdings_type='serial',
                 patterns=patterns)
             click.echo(
```

Comment:
If this is for logging we shall use:
`current_app.logger`

Reply:
not for logging